tt_eager/tt_lib/csrc/tt_lib_bindings_tensor_impl.hpp

```diff
@@ -5,12 +5,14 @@
 #pragma once
 
 #include "tt_lib_bindings.hpp"
-#include "tt_dnn/op_library/eltwise_unary/eltwise_unary_op.hpp"
+#include "ttnn/operations/eltwise/unary/device/unary_op.hpp"
 
 namespace tt::tt_metal{
 
 namespace detail {
 
+using ttnn::operations::unary::UnaryWithParam;
```

Comment:
I recommend to remove this. This file has a single use of UnaryWithParam and it is better to just use the full one.
The problem with this using is that it is in the header so not every file including this header will have UnaryWithParam directly inside `tt::tt_metal::detail`.

Reply:
removed and used ttnn::operations::unary::UnaryWithParam directly. Thanks.